src/store/epics/utils.ts

```diff
@@ -8,3 +8,14 @@ export const withPolling = (delay: number, interval?: number) => pipe(
     action => timer(delay, interval).pipe(mapTo(action))
   )
 )
+
+export const failureAttemptHandler = (timestamps, timeNow) => {
```

Comment:
I'd use Date instead of number since it's clearer what the value is supposed to be then.

Reply:
done